packages/sdk/docs/README.md

```diff
@@ -0,0 +1,51 @@
+@namada/sdk / [Exports](modules.md)
+
+# sdk
+
+Namada SDK package
+
+## Development
+
+```bash
+# Build wasm for single core, release mode
+yarn wasm:build[:node]
+
+# Build wasm for multicore, release mode
+yarn wasm:build[:node]:multicore
+
+# Build wasm with debugging for single core
+yarn wasm:build[:node]:dev
+
+# Build wasm with debugging for multicore
+yarn wasm:build[:node]:dev:multicore
+```
+
+## Usage
+
+```typescript
+import { Sdk } from "@namada/sdk"
```

Comment:
Does this example still need updated?

Reply:
I've updated README here https://github.com/anoma/namada-interface/pull/695/files :)